staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -835,6 +835,17 @@ type CDISpec struct {
 	PriorityClass *CDIPriorityClass `json:"priorityClass,omitempty"`
 }
 
+type ComponentConfig struct {
+	// NodePlacement describes scheduling configuration for specific CDI components
+	sdkapi.NodePlacement `json:"nodePlacement,omitempty"`
```

Comment:
Again, we want to embed the fields so just : `sdkapi.NodePlacement` no json tags.

as-as, the json path is `spec.infra.nodePlacement.nodeSelector` for example

Ant that is not compatible with current json path `spec.infra.nodeSelector`

Reply:
It appears that we don't need to modify the cdi-cr yaml if sdkapi.NodePlacement has no json tags, thank you for your interpretation.